gprofiler/profiler_base.py

```diff
@@ -31,10 +38,40 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.stop()
 
 
-class NoopProfiler(ProfilerBase):
+class ProfilerBase(ProfilerInterface):
```

Comment:
ABC?

Reply:
We don't need anything that `ABCMeta` provides